mathesar/api/ui/viewsets/users.py

```diff
@@ -16,6 +21,16 @@ class UserViewSet(AccessViewSetMixin, viewsets.ModelViewSet):
     pagination_class = DefaultLimitOffsetPagination
     access_policy = UserAccessPolicy
 
+    @action(methods=['post'], detail=False)
```

Comment:
Why not make this a detail view? You don't need the username then.

Reply:
That makes sense if the front end is listing users and knows about the `id`. I was thinking of it as a form for the superuser to enter the username of the user. I will change it to a detail view